update.sh

```diff
@@ -299,8 +302,8 @@ print_expose_ports() {
 	cat >> $1 <<-EOI
 	# $expose_comment
 	EXPOSE $expose_ports
-
-	EOI
+	
```

Comment:
And remove this tab because it's not used for anything.

Reply:
Fixed